presto-tpch/src/main/java/io/prestosql/plugin/tpch/TpchMetadata.java

```diff
@@ -197,17 +195,6 @@ public ConnectorTableHandle getTableHandleForStatisticsCollection(ConnectorSessi
                 .collect(toSet());
     }
 
-    @Override
-    public ConnectorTableLayout getTableLayout(ConnectorSession session, ConnectorTableLayoutHandle handle)
```

Comment:
Should this be part of the `Return TPCH table predicate when no applyFilter was called` commit?

Looks like a removal of obsolete method after `Remove table layouts from TPCH connector` commit.
If correct, extract to obsolete commit
```
Remove obsolete override

TPCH doesn't use legacy TableLayouts since c1bdbd7149f35e11abec7c48b964d1e765e3c205.
```